assets/src/scripts/network/components/network-sites/index.spec.js

```diff
@@ -25,43 +27,55 @@ describe('network map module', () => {
     });
 
     afterEach(() => {
-        select('#map').remove();
+        container.remove();
         jasmine.Ajax.uninstall();
     });
 
 
     describe('Map creation without Network maps', () => {
         beforeEach(() => {
             store = configureStore();
-            attachToNode(store, mapNode, {
+            spyOn(Actions, 'retrieveNetworkData').and.returnValue(function () {
+                return Promise.resolve({});
+            });
+            attachToNode(store, componentNode, {
                 networkcd: 'AHS',
                 extent: '[-93.078075, 34.513375, -92.986325, 34.588425]'
             });
-        });
 
-        it('Should create a leaflet map within the mapNode with', () => {
-            expect(select(mapNode).selectAll('.leaflet-container').size()).toBe(1);
+
         });
 
-        it('Should not create an overlay layer', () => {
-            expect(select(mapNode).selectAll('.leaflet-overlay-pane img').size()).toBe(0);
+        it('Should create a leaflet map within the mapNode with', () => {
+            expect(select(componentNode).selectAll('.leaflet-container').size()).toBe(1);
         });
 
         it('Should create a legend control', () => {
-            expect(select(mapNode).selectAll('.legend').size()).toBe(1);
+            expect(select(componentNode).selectAll('.legend').size()).toBe(1);
         });
 
-        it('Should create not create Network Legend', () => {
-            expect(select(mapNode).select('#network-legend-list').size()).toBe(0);
+        it('Should not create an overlay layer', (done) => {
+            window.requestAnimationFrame(() => {
+                expect(select(componentNode).selectAll('.leaflet-overlay-pane img').size()).toBe(0);
+                done();
+            });
         });
 
-        it('Should create a leaf-control-layers class', () => {
-            expect(select(mapNode).selectAll('.leaflet-control-layers').size()).toBe(1);
+        it('Should create not create Network Legend', (done) => {
```

Comment:
It should say "Should not create Network Legend", I'm really correcting myself, ha.